src/test/suite/connectionController.test.ts

```diff
@@ -1,10 +1,11 @@
 import * as sinon from 'sinon';
+import * as util from 'util';
```

Comment:
Is there a reason we want to use this format for importing? 
`import { promisify } from 'util';` looks a little cleaner imo - I'm cool with either.

Reply:
I transformed it to debug like this `console.log(`${util.inspect(error)}`)` and kept it this way so in the future I could do it again.